person_capture/gui_app.py

```
@@ -5085,6 +5413,12 @@ def save_hit(c, idx):
                     )
 
                     processed_crop_xyxy = (int(cx1), int(cy1), int(cx2), int(cy2))
+                    try:
+                        final_crop_for_sharp = frame[int(cy1):int(cy2), int(cx1):int(cx2)]
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `frame`

(B023)